superset/views/core.py

```diff
@@ -2710,6 +2711,11 @@ def sql_json(self):
                     rendered_query,
                     return_results=True,
                     user_name=g.user.username if g.user else None)
+
+            display_limit = app.config.get('DISPLAY_MAX_ROW', None)
```

Comment:
Could we refactor this logic into a function as it seems to be identical as the logic above.

Reply:
Done